pipenv/core.py

```diff
@@ -678,6 +679,12 @@ def _cleanup_procs(project, procs, failed_deps_queue, retry=True):
                 failed_deps_queue.put(dep)
 
 
+def get_trusted_hosts():
+    trusted_hosts = []
+    trusted_hosts.extend(os.environ.get("PIP_TRUSTED_HOSTS", []))
+    return trusted_hosts
```

Comment:
PIP_TRSUTED_HOSTS is a space separated list, so we should do here:
```
try:
    return os.environ.get("PIP_TRUSTED_HOSTS", []).split(" ")
execpt AttributeErorr:
    return []
```
```
And we can also remove line 683.

Reply:
This was the existing code just relocated, but perhaps there was a bug with it?

Reply:
Pushed the refactor change.